tests/test_gauss_pdf_and_cdf.py

```diff
@@ -0,0 +1,41 @@
+from os.path import exists
+from src.gauss_pdf_and_cdf import (
+    GaussianProbabilisticSetting,
+    BetaProbabilisticSetting
+)
+import cv2
+
+
+def examine_show_or_save_figure_function(distribution_type: str) -> None:
+    '''Test show_figure, save_figure function.
+
+    Args:
+        distribution_type: 'gaussian' or 'beta'
+    '''
+    if distribution_type == 'gaussian':
+        distribution = GaussianProbabilisticSetting()
+    elif distribution_type == 'beta':
+        distribution = BetaProbabilisticSetting()
+    else:
+        raise NotImplementedError()
+
+    lengths = distribution.show_figure()
+
+    for length in lengths:
```

Comment:
Write assert statement in __init__ function, because people cannot understand what x and y mean, and where they are defined seeing an error message here.

Reply:
Can I define an init function in the test file?

Reply:
I wrote assert statement in distibution_function.py.